Cargo.toml

```diff
@@ -12,7 +12,7 @@
 
 [package]
 name = "hotshot-query-service"
-version = "0.1.23"
+version = "0.1.24"
```

Comment:
This should be 0.1.27 as we have a tag for 0.1.26

Reply:
[36db5de](https://github.com/EspressoSystems/hotshot-query-service/pull/601/commits/36db5decbcbe1cfce78c720513d946b2535989ca)